---
layout: paper
title: "Topological patterns in microRNA–gene regulatory network: studies in colorectal and breast cancer"
year: "2013"
shortref: "Sengupta et al. Molecular bioSystems 2013"
nickname:
journal: "Molecular bioSystems"
volume: 9
issue: 6
pages: 1360-1371
authors: "Debarka Sengupta, Sanghamitra Bandyopadhyay"
image: https://debarka.github.io/senguptalab/assets/images/papers/TopologicalPatterns.png
pdf:
pdflink: http://pubs.rsc.org/en/Content/ArticleLanding/2013/MB/c3mb25518b#!divAbstract
github:
pmid: 23475160
pmcid:
f1000:
figshare:
doi: 10.1039/c3mb25518b.
category: paper
published: true
peerreview:
review: false
tags:
---
{% include JB/setup %}


# Abstract

It is now widely accepted that microRNAs (miRNAs or miRs) along with transcription factors (TFs) weave a complex inter-regulatory network within the cell that is responsible for the combinatorial regulation of gene expression. Recently we have shown that miRNAs and TFs that form network clusters are also associated with a number of common diseases. However, the quest persists to find out topological structures that facilitate disease progression. In the current work we choose colorectal and breast cancers for our analysis. For this, the human genome wide TF–miRNA–gene network (TMG-net) is first built by combining experimentally validated and confidently predicted miRNA → gene (including TF genes), TF → gene and TF → miRNA interactions. Subnetworks active in colorectal and breast cancers are extracted from the TMG-net and then analyzed. Disease specific subnetworks are found to be significantly dense, having a pyramid shaped hierarchical backbone of interactions. Interestingly, most of the top level molecules (e.g., hsa-mir-210, hsa-mir-378) are found to be already established as oncomirs. TFs that are dysregulated in a particular cancer, are found to be well-linked via miRNAs and other TFs, with miRNAs being highly predominant. Analogous to density, a new measure called Inductive Converge (InCov) is proposed and used to analyze the natural association of molecules in the disease specific networks. Finally a web application called DisTMGneT (Disease Specific TF–miRNA–gene Network) is developed for disease specific subnetworks from the TMG-net, based on user supplied sets of dysregulated miRNAs, TFs and non TF genes. DisTMGneT is available at http://www.isical.ac.in/bioinfo_miu/dscsgen.php.
