---
layout: paper
title: "FORKS: Finding Orderings Robustly using k-means and Steiner trees"
year: "2017"
shortref: "Debarka et al. bioRxiv 2017"
nickname:
journal: "bioRxiv"
volume:
issue:
pages: 132811
authors: "Mayank, Huipeng Li, Debarka Sengupta, Shyam Prabhakar, Jayadeva Jayadeva"
image: https://debarka.github.io/senguptalab/assets/images/papers/FORKS.png
pdf:
pdflink: https://www.biorxiv.org/content/early/2017/06/20/132811
github:
pmid:
pmcid:
f1000:
figshare:
doi: https://doi.org/10.1101/132811
category: paper
published: true
peerreview:
review: false
tags:
---
{% include JB/setup %}


# Abstract

Recent advances in single cell RNA-seq technologies have provided researchers with unprecedented details of transcriptomic variation across individual cells.However, it has not been straightforward to infer differentiation trajectories from such data, due to the parameter-sensitivity of existing methods. Here, we present Finding Orderings Robustly using k-means and Steiner trees (FORKS), an algorithm that pseudo-temporally orders cells and thereby infers bifurcating state trajectories. FORKS, which is a generic method, can be applied to both single-cell and bulk differentiation data. It is a semi-supervised approach, in that it requires the user to specify the starting point of the time course. We systematically benchmarked FORKS and eight other pseudo-time estimation algorithms on six benchmark datasets, and found it to be more accurate, more reproducible, and more memory-efficient than existing methods for pseudo-temporal ordering. Another major advantage of our approach is its robustness âˆ’ FORKS can be used with default parameter settings on a wide range of datasets.
